### ISSUE TITLE
The oauth2login samples should show OIDC logout configuration

### ISSUE THREAD

**Issue by jzheaux** [spring-projects/spring-security-samples#3]
Related to https://github.com/spring-projects/spring-security/issues/7285#issuecomment-523240651

**Comment by ThomasVitale** [comment]
@jzheaux Yes, I can do that. Should I use only one OIDC provider in the sample? For example, would Keycloak be ok? Any recommendation for the name of the project (that I guess should go in `spring-security-samples/servlet/spring-boot/java/oauth2`).

**Comment by jzheaux** [comment]
Yep, I think that one provider would be fine. For now, it might be best to leave placeholders for the properties, similar to how the  oauth2Login samples do it. Since Okta offers free dev accounts and they support OIDC, you could borrow the other samples' Okta instructions for the README.

For a name, I'd do `oidc/login`, as in `spring-security-samples/servlet/spring-boot/java/oidc/login`.

**Comment by ThomasVitale** [comment]
Perfect, thank you. I'll do that.

**Comment by erycoking** [comment]
Hi guys, am facing an issue with the spring oauth2 authorization server. Once the user signs in for the first time using the authorization code flow, the next time they try to log in they are automatically logged in without filling in the username and password, which is not the desired behavior for me. Is there a way to log them out completely so that they will have to fill in the form again to log in?? please any help would go a long way? I am stuck, I have also posted this question on stack overflow, you can follow this link [https://stackoverflow.com/questions/70735327/spring-oauth2-authorization-server-unable-to-logout-users](https://stackoverflow.com/questions/70735327/spring-oauth2-authorization-server-unable-to-logout-users)
